stretch/src/bankers.c

```diff
@@ -90,7 +90,7 @@ int get_random_amount(void)
 	// !!!! IMPLEMENT ME:
```

Comment:
- we can certainly go the other way and use it for **deposits**
- also, instead of generating a random number, we can get the **previous balance**
- instead of writing to console, we can write to a ledger OR account reconciliation against account payable/receivable
- we can also read the input from a bank download and populate a database via API
- on a more simpler note, we can color the console output (blue for withdrawals /green for deposits / red for overdraft and yellow for bank fees.
- we can wrap this in a DLL and call it via bash in a Python back-end.
- etc ...

Reply:
Fantastic response! 🥇